html_css/html5_basics.md

```diff
@@ -26,4 +26,5 @@ This section contains helpful links to other content. It isn't required, so cons
 * [WTFHTMLCSS with basic tips to improve your code](http://wtfhtmlcss.com/)
 * [W3C HTML5 tutorial ](http://www.w3schools.com/html/default.asp)
 * [HTML Style Guide](http://codeguide.co/#html) by [Mark Otto](https://github.com/mdo), one of the creators of Bootstrap.
+* [CrashCourse Introduction to HTML&CSS] (https://www.youtube.com/watch?v=QA0XpGhiz5w)
```

Comment:
![image](https://user-images.githubusercontent.com/4411701/76137857-49f43480-6007-11ea-9075-a6873f314867.png)
This markup is invalid, please fix and I'll merge :)